Sources/Raylib/raylib_h_text.swift

```diff
@@ -29,11 +29,15 @@ public extension Raylib {
     
     /// Load font from file with extended parameters
     @inlinable
-    static func loadFontEx(_ fileName: String, _ fontSize: Int32, _ fontChars: [Character]) -> Font {
+    static func loadFontEx(_ fileName: String, _ fontSize: Int32, _ fontChars: [Character]?) -> Font {
```

Comment:
Let's add a default `nil` value (`fontChars: [Character]? = nil`) here so SourceKit can autocomplete this function without the fontChars argument. That'll make the default behavior more obvious to users.

Reply:
I agree, I've added the default value. Thanks!